src/coreclr/tools/Common/TypeSystem/Interop/IL/MarshalHelpers.Aot.cs

```diff
@@ -135,5 +135,25 @@ public static MethodIL EmitExceptionBody(string message, MethodDesc method)
 
             return new PInvokeILStubMethodIL((ILStubMethodIL)emitter.Link(method), isStubRequired: true);
         }
+
+        public static MethodIL EmitExceptionBody(Exception exception, MethodDesc method)
```

Comment:
I do not think this should be needed. Incorrectly resolved merge conflict?

Reply:
Yeah. That was left-over when my initial attempt :(